assets/src/dashboard/components/navigationBar/components.js

```diff
@@ -22,21 +22,29 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
+import { Z_INDEX } from '../../constants';
 import Button from '../button';
 
 export const Nav = styled.nav`
   ${({ theme }) => `
-    position: relative;
+    position: fixed;
```

Comment:
I think we need to make this an absolutely positioned element over the scrollable content. The only issue with `position: fixed` elements are that they're spaced relative to the viewport which kind of throws off our app being contained to a shell. 

For instance, since this is `position: fixed` & `width: 100%` it's 100% of the viewport width. If it was relative/absolute, it would be 100% of it's nearest position relative/absolute ancestor.

This is made more apparent if you uncollapse the WP toolbar:
<img width="1440" alt="Screen Shot 2020-05-04 at 10 54 01 AM" src="https://user-images.githubusercontent.com/35983235/80991971-13178080-8df6-11ea-9b7a-bd3777775244.png">

I would recommend leaving this `position: relative`, and on the same view just do:
```
<Layout.Provider>
  <Layout.Fixed>
    // put relatively positioned header in here
  </Layout.Fixed>
  <Layout.Scrollable>
    // Put scroll content in here
  </Layout.Scrollable>
</Layout.Provider>
```
which should take care of this for you. (The provider isn't really needed, but the subcomponents use a hook that throws an error if it doesn't have a provider)

Reply:
great idea! For some reason i was completely overlooking the Layout.Fixed as an option for the nav. This eliminates the need to do anything special to the detail nav.